--- a/optifine/xdelta/PatchException.java
+++ b/optifine/xdelta/PatchException.java
@@ -1,13 +1,10 @@
 package optifine.xdelta;
 
-public class PatchException extends Exception
-{
-    public PatchException()
-    {
+public class PatchException extends Exception {
+    public PatchException() {
     }
 
-    public PatchException(String msg)
-    {
+    public PatchException(String msg) {
         super(msg);
     }
 }
