--- a/com/mojang/realmsclient/gui/screens/NotifableRealmsScreen.java
+++ b/com/mojang/realmsclient/gui/screens/NotifableRealmsScreen.java
@@ -1,10 +1,10 @@
 package com.mojang.realmsclient.gui.screens;
 
 import com.mojang.realmsclient.dto.WorldTemplate;
-import javax.annotation.Nullable;
 import net.minecraft.realms.RealmsScreen;
 
-public abstract class NotifableRealmsScreen extends RealmsScreen
-{
+import javax.annotation.Nullable;
+
+public abstract class NotifableRealmsScreen extends RealmsScreen {
     protected abstract void func_223627_a_(@Nullable WorldTemplate p_223627_1_);
 }
