--- a/com/mojang/realmsclient/exception/RealmsHttpException.java
+++ b/com/mojang/realmsclient/exception/RealmsHttpException.java
@@ -1,9 +1,7 @@
 package com.mojang.realmsclient.exception;
 
-public class RealmsHttpException extends RuntimeException
-{
-    public RealmsHttpException(String p_i51786_1_, Exception p_i51786_2_)
-    {
+public class RealmsHttpException extends RuntimeException {
+    public RealmsHttpException(String p_i51786_1_, Exception p_i51786_2_) {
         super(p_i51786_1_, p_i51786_2_);
     }
 }
